radio/src/gui/480x272/model_select.cpp

```diff
@@ -108,6 +113,7 @@ void onModelSelectMenu(const char * result)
   else if (result == STR_DELETE_MODEL) {
     POPUP_CONFIRMATION(STR_DELETEMODEL);
     SET_WARNING_INFO(currentModel->name, LEN_MODEL_FILENAME, 0);
+    deleteMode = MODE_DELETE_MODEL;
```

Comment:
Yes no choice, but we really need to change that in the near future with a function pointer!